CMakeLists.txt

```diff
@@ -111,6 +111,87 @@ else()
     message(FATAL_ERROR "Unknown OS type")
 endif()
 
+if(NOT UMF_BUILD_LIBUMF_POOL_JEMALLOC)
+    message(STATUS "jemalloc is disabled")
```

Comment:
I guess we don't have any such msg when an option is disabled

Reply:
Replaced with `set(UMF_POOL_JEMALLOC_ENABLED FALSE)`